src/gbxml/ReverseTranslator.cpp

```diff
@@ -85,16 +85,34 @@ namespace gbxml {
 
     if (openstudio::filesystem::exists(path)) {
 
-      // validate the gbxml prior to reverse translation
-      auto gbxmlValidator = XMLValidator::gbxmlValidator();
-      gbxmlValidator.validate(path);
-
       openstudio::filesystem::ifstream file(path, std::ios_base::binary);
       if (file.is_open()) {
         pugi::xml_document doc;
         auto load_result = doc.load(file);
         if (load_result) {
-          result = this->convert(doc.document_element());
+          auto root = doc.document_element();
+          if (std::string_view{root.name()} != "gbXML") {
+            LOG(Error, "Expected the root element to be <gbXML>");
+            return boost::none;
+          }
+          // Scan version of gbxml schema
+          std::string version = root.attribute("version").value();
+          if (version.empty()) {
+            LOG(Warn, "gbXML has no `version` attribute for the schema version, assuming 7.03.");
+            version = "7.03";
+          }
+          if (version == "7.03") {
```

Comment:
Only suggestion I would have is to put "7.03" in `XMLValidator` since that is the thing that knows which version of the schema it has.  Then if `version != 7.03`, a call to `gbxmlValidator.validate` would log something like `Cannot validate #{version}`.

Reply:
Or keep the code as is but return "7.03" from a `XMLValidator::gbxmlValidatorExpectedVersion()` method or something

Reply:
Would ccffa3747232baa9e364e28514b47e9e6d26de74 work? @macumber

Reply:
Would ccffa3747232baa9e364e28514b47e9e6d26de74 work? @macumber